### PR TITLE
Create we_accept_monero_es.md

### PR DESCRIPTION
**Pull request by dilubit**
This is a spanish translation from https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/en/archive/we_accept_monero_en.md

### CODE REVIEW

File: monero-outreach-docs/translations/es/archive/we_accept_monero_es.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Aceptamos Monero
+
+**“¿Estás cansado de implorar por adopción? ¿Harto de esperar el amanecer de Monero? Se parte del cambio que quieres ver en el ecosistema.” - 20th of MAy, 2020**
+
+Nos gustaría invitarte a unirte a nuestros esfuerzos para llevar a cabo una buena 'campaña agobiante' a la antigua con tus comerciantes favoritos que aún no aceptan XMR. La 'campaña agobiante' más impresionante y exitosa de todos los tiempos tuvo que haber sido escrita por George Lois. En 1982, las empresas de cable no tenían interés en un canal de música las 24 horas, no creían que funcionaría. A George Lois se le ocurrió una idea y filmó una serie de comerciales en los que las estrellas de rock pedían a los espectadores que llamaran a su compañía de cable local y dijeran "¡Quiero mi MTV!" El resto es historia, [youtu.be/vAyOcWpm3QU](https://youtu.be/vAyOcWpm3QU?t=71).
```
Comment (quagmir3) [inline monero-outreach-docs/translations/es/archive/we_accept_monero_es.md:5]: cambiaría en este párrafo el término "compañía de cable local" por "operadora de televisión local"

**Review by quagmir3 — CHANGES_REQUESTED** [review]
Está todo bien. He puesto unos comentarios diciendo algunas cosas que cambiaría, pero veo bien el resto.

**Comment by dilubit** [comment]
Estoy de acuerdo con los comentarios de @quagmir3

**Review by dilubit — COMMENTED** [review]
I agree with the @quagmir3 reviews

**Review by dilubit — COMMENTED** [review]
I updated the changes suggested by @quagmir3

**Comment by dilubit** [comment]
I updated the changes suggested by @quagmir3

**Comment by quagmir3** [comment]
Todo correcto

File: monero-outreach-docs/translations/es/archive/we_accept_monero_es.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
 
-La misión, si decides aceptarlo, es comunicarte con tu comerciante favorito y preguntarle si ha considerado aceptar Monero, porque ese sería su método de pago preferido. Puede que digan que sí, que digan que no, pero al menos les has hecho pensar en ello. No será la última vez que escuchen sobre Monero, pero podría ser la primera.
+La misión, si decides aceptarlo, es comunicarte con tu comerciante favorito y preguntarle si ha considerado aceptar Monero, porque ese sería tu método de pago preferido. Puede que digan que sí, que digan que no, pero al menos les has hecho pensar en ello. No será la última vez que escuchen sobre Monero, pero podría ser la primera.
```
Comment (lh1008) [inline monero-outreach-docs/translations/es/archive/we_accept_monero_es.md:7]: I have a suggestion. 

"La misión, si decides **acpetarla**,..."

**Review by lh1008 — COMMENTED** [review]
My comment is because "la misión" is feminine in this case. :)

File: monero-outreach-docs/translations/es/archive/we_accept_monero_es.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+En marketing se dice que alguien necesita ver un mensaje siete veces antes de darse cuenta de él. Esa es una regla general para la publicidad, y esperamos fomentar una conversación con los comerciantes. Una buena conversación vale fácilmente siete anuncios publicitarios. La investigación ha encontrado que las personas que conocen una organización tienen cinco veces más probabilidades de tener una opinión favorable de ella. En otras palabras, cuanto más hables de Monero, a más gente le gustará.
+
+También hemos creado este folleto del tamaño de un billete de un dólar que puedes imprimir y guardar en tu billetera cuando estés en la calle. Este artículo se centra en las tres características únicas de Monero que son las más importantes para los propietarios de pequeñas empresas y los invita a aprender más en nuestras páginas de [Monero Merchant Services] (https://www.monerooutreach.org/merchants/).
```
Comment (lh1008) [inline monero-outreach-docs/translations/es/archive/we_accept_monero_es.md:11]: Please remove the space between the square brackets [] and the normal brackets ().

[Monero Merchant Services] (https://www.monerooutreach.org/merchants/)

[Monero Merchant Services](https://www.monerooutreach.org/merchants/)

**Review by lh1008 — CHANGES_REQUESTED** [review]
Please remove the space between [ ] ( ) :).

File: monero-outreach-docs/translations/es/archive/we_accept_monero_es.md (hunk 2 above)
Comment (dilubit) [reply monero-outreach-docs/translations/es/archive/we_accept_monero_es.md:7]: ok

File: monero-outreach-docs/translations/es/archive/we_accept_monero_es.md (hunk 3 above)
Comment (dilubit) [reply monero-outreach-docs/translations/es/archive/we_accept_monero_es.md]: Done!

File: monero-outreach-docs/translations/es/archive/we_accept_monero_es.md (hunk 2 above)
Comment (dilubit) [reply monero-outreach-docs/translations/es/archive/we_accept_monero_es.md]: Done, It was fixed

File: monero-outreach-docs/translations/es/archive/we_accept_monero_es.md (hunk 3 above)
Comment (dilubit) [reply monero-outreach-docs/translations/es/archive/we_accept_monero_es.md]: The space was removed

**Review by lh1008 — APPROVED** [review]
Everything looks good. Thank you @dilubit and @quagmir3.

:)

**Pull request merged**